prototypes/stats/frontend/tailwind.config.ts

```diff
@@ -14,6 +14,29 @@ const config: Config = {
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
       },
     },
+    colors: {
+      'osrslb': {
```

Comment:
oh neat, so this configuration here generates the `bg-osrslb-100` kinda css classes?

Reply:
Yep! anything that uses 'color - bg-osrslb-100, text-osrslb-100, border-osrslb-100 etc.